features/step_definitions/basic_steps.rb

```diff
@@ -1,7 +1,21 @@
 When(/^I visit "([^"]*)" page$/) do |page_name|
-  visit root_path
+  visit photos_path
```

Comment:
You need to fix this step definition to handle visiting any pages. Once you do, all the tests will go green again.